client/deliveries.lua

```diff
@@ -52,7 +52,7 @@ local function KnockDoorAnim(home)
     local knockAnimLib = "timetable@jimmy@doorknock@"
     local knockAnim = "knockdoor_idle"
     local PlayerPed = PlayerPedId()
-    local myData = QBCore.Functions.GetPlayerData()
+    local myData = QBX.PlayerData
```

Comment:
This seems odd to me. Should references to myData instead be replaced with QBX.PlayerData?